### ISSUE TITLE
Replace obsolete MOVE for chained fields

### ISSUE THREAD

**Issue by jelliottp** [SAP/abap-cleaner#32]
Replacing the obsolete `MOVE` command is a great rule! However it does not seem to work for multiple chained fields.

Before
```
MOVE:
  source1 TO destination1,
  source2 TO destination2.
```
After
```
destination1 = source1.
destination2 = source2.
```

**Comment by ConjuringCoffee** [comment]
It works correctly if the rule "Unchain into multiple statements" is active too:

![image](https://github.com/SAP/abap-cleaner/assets/72548231/cfba6c34-059e-46a7-bcb2-ffb01686e6fc)

**Comment by jmgrassau** [comment]
Hi Josh, 

yes, those chain colons are a bit of a pain, because you could have them in any place, making cleanup rules' lives more difficult! E.g., you might have

```ABAP
  ADD 1 TO: lv_any, 
            lv_other.
```

But I admit that MOVE: could be a particularly common finding, so I guess it is worth enhancing the "Replace obsolete MOVE …" rule accordingly (even if the "Unchain" rule is deactivated). 

Kind regards,
Jörg-Michael

**Comment by jelliottp** [comment]
> It works correctly if the rule "Unchain into multiple statements" is active too:
> 
> ![image](https://user-images.githubusercontent.com/72548231/241206378-dbdf1e9e-3b7e-4826-8356-82925cb67ee8.png)

Interesting, I am using that rule and that wasn't my experience. However I looked and this particular setting was not on:
![image](https://github.com/SAP/abap-cleaner/assets/63249342/47f43209-5b1b-4606-8b5d-6dde8f08ae2f)

Once that is enabled it does work. Even with that said, because this is an obsolete statement I think the default behavior of the "Replace obsolete MOVE …" rule should also handle this.

**Comment by jmgrassau** [comment]
Hi Josh, 

this new option is now available with version 1.3.0!

![image](https://github.com/SAP/abap-cleaner/assets/79639298/3bee8805-f4e1-483f-b21a-f5bc5717a1e0)

So MOVE: chains can now be replaced even if "Unchain into multiple statements" is deactivated.

Kind regards,
Jörg-Michael
